lib/plugins/output-filter/kubernetes-enrichment.js

```diff
@@ -0,0 +1,212 @@
+const { KubeConfig } = require('kubernetes-client')
+const Client = require('kubernetes-client').Client
+const Request = require('kubernetes-client/backends/request')
+const kubeconfig = new KubeConfig()
+var consoleLogger = require('../../util/logger.js')
+var podCache = {}
+var digestRegEx = /sha256:.+/i
+var client = null
+const FALSE_REGEX = /false/i
+if (process.env.KUBERNETES_PORT_443_TCP !== undefined) {
```

Comment:
Can you use `if (process.env.KUBERNETES_PORT_443_TCP) {` instead?

Reply:
I've got used to undefined checks as in some situations a value of "" or 0 or "0" could lead to wrong decisions. In the specific case (where we probably will not find port 0) it might be ok ... - trust me I had fix N times bugs in 'if' statements with ```if (someNonBooleanVar) ... ``` so i think it is a good practice to use `if (someNonBooleanVar !== undefined)